ocs_ci/templates/ocs-deployment/deploy-with-olm.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: v1
```

Comment:
why static files again? instead we should point to the files directly from the github repo's to avoid out of sync.

Reply:
I discussed this with Kaushal on Friday and he told me that there are no planed changes in this file. Also based on comment from Jose in this bz:  https://bugzilla.redhat.com/show_bug.cgi?id=1770290#c2 he wrote:
> For this and the above, there is no need to make the example CR in GitHub match the CR that is created by the UI. QE should be able to (and possibly should be in general) creating their own CR rather than using the example in the repo.

So as I understand we should not rely on their examples.  In this deploy with OLM file, for example we were based on master branch and for example my recent PR: 
https://github.com/openshift/ocs-operator/pull/271
Will not be cherrypicked to 4.2 release branch which we should probably go with so I decided to have own file in our repo for 4.2. Later on we can maybe reuse this file again from the master, but currently I was having the issue with defined local-storage stuff in this file:
https://github.com/openshift/ocs-operator/blob/release-4.2/deploy/deploy-with-olm.yaml#L9-L53 .

There will need to be changes also for stage CDN deployment I guess.

If I would follow the requirements in the docs for UI deployment only one relevant in this file is namespace, the rest is just workaround to get custom registry which provide internal builds to us.

Reply:
Petr is correct, the example files in the GitHub repo should not be taken as accurate for any given downstream release of OCS.

Reply:
Resolving.